spec/controllers/vm_cloud_controller_spec.rb

```diff
@@ -56,6 +61,33 @@
       expect(response.body).to_not be_empty
     end
 
+    it 'can open instance resize tab' do
+      post :explorer
+      expect(response.status).to eq(200)
+
+      post :x_button, :params => {:pressed => 'vm_protect', :id => vm_openstack.id}
+      expect(response.status).to eq(200)
+
+      allow(controller).to receive(:x_node).and_return("v-#{vm_openstack.compressed_id}")
+
+      post :x_button, :params => {:pressed => 'instance_resize', :id => vm_openstack.id}
+      expect(response.status).to eq(200)
+      expect(response).to render_template(:partial => 'vm_common/_resize')
+    end
+
+    it 'can resize an instance' do
+      flavor = FactoryGirl.create(:flavor_openstack)
+      controller.instance_variable_set(:@edit,
+                                       :new      => {:flavor => flavor.id},
+                                       :explorer => true)
+      expect_any_instance_of(VmCloud).to receive(:resize).with(flavor)
+      post :resize_vm, :params => {
+        :button => 'submit',
+        :id     => vm_openstack.id
+      }
+      expect(response.status).to eq(200)
```

Comment:
@mansam  i was under the impression that we can probably check flavor being updated after instance is resized.
@dclarizio looks good, ready to merge.